vignettes/packages.bib

```diff
@@ -28,7 +28,7 @@ @Manual{R-OBIC
   author = {Sven Verweij and Gerard Ros and Yuki Fujita and Brent Riechelman},
   year = {2022},
   note = {R package version 2.0.1},
-  url = {https://CRAN.R-project.org/package=OBIC},
+  url = {https://github.com/AgroCares/Open-Bodem-Index-Calculator},
```

Comment:
Please leave this to CRAN

Reply:
resolved in 967da56ec7cbbd9010713958cb5286ad8ec7d541